nixos/modules/services/backup/borgbackup.nix

```diff
@@ -92,6 +90,7 @@ let
         PrivateTmp = cfg.privateTmp;
       };
       environment = {
+        BORG_BASE_DIR = userHome;
```

Comment:
This should be added to `mkBorgWrapper`, though I really don't like that duplication. Neither do I like `mkBorgWrapper` at all.

Reply:
Yeah, then we have to duplicate `userHome` logic.

Reply:
Actually, we should only set `environment.BORG_BASE_DIR` if using a dynamic user.

Reply:
Done.